testData/solutions/win32.json

```diff
@@ -1428,5 +1428,139 @@
         ],
         "start": [],
         "stop": []
+    },
+
+    "eu.gpii.olb": {
+        "name": "Online Banking Prototype",
+        "contexts": {
+            "OS": [
+                {
+                    "id": "win32"
+                }
+            ]
+        },
+        "settingsHandlers": {
+            "conf": {
+                "type": "gpii.settingsHandlers.webSockets",
+                "options": {
+                    "path": "gpii.eu/olb/"
```

Comment:
Oh, I see. I misread something that I had seen in your pull request for the Cloud4all site. Sorry.